src/css-loader.js

```diff
@@ -1,28 +1,50 @@
 module.exports = c => c;
 
+// createLoaderObject: https://github.com/webpack/loader-runner/blob/master/lib/LoaderRunner.js
+function createLoader(loader) {
+  // const query = loader.ident ? `??${loader.ident}` : '';
```

Comment:
?

Reply:
Still trying to figure this out. I know you are allowed to write to this.loaders but there is some magic that doesn't work with things handled by importloaders via css-loader...the answer is deep in webpack arcana